### PR TITLE
Add Sync Server Recordings task

### PR DESCRIPTION
**Pull request by scouillard**
Add Rake Task to Sync the Server Recordings.

### CODE REVIEW

**Comment by scouillard** [comment]
RecordingsSync (service) and ServerRecordingsSync (rake task) share similar logic.  But I think it is different enough to warrant the duplication.

File: lib/tasks/server_recordings_sync.rake [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+# frozen_string_literal: true
+
+desc 'Server Recordings sync with BBB server'
+
+task server_recordings_sync: :environment do
+  Recording.destroy_all
+
+  Room.select(:id, :meeting_id).in_batches(of: 25) do |rooms|
+    meeting_ids = rooms.pluck(:meeting_id)
```
Comment (farhatahmad) [inline lib/tasks/server_recordings_sync.rake:25]: Can you not just do 
```
 Room.select(:meeting_id).in_batches(of: 25) do |rooms|
```

Comment (scouillard) [reply lib/tasks/server_recordings_sync.rake:25]: No, all of the "batches" methods seem to return an ActiveRecord object.
Rails does not seem to provide a built-in way to [batch and pluck](https://gist.github.com/iamatypeofwalrus/f9f6a3049f63dd05d02c)

Comment (scouillard) [reply lib/tasks/server_recordings_sync.rake:25]: There is a way to return the ids/index without firing an extra query https://stackoverflow.com/a/66885913

**Review by farhatahmad — APPROVED** [review]

**Pull request merged**